crates/ruma-identifiers/src/user_id.rs

```diff
@@ -100,7 +103,7 @@ impl UserId {
     /// A historical user ID is one that doesn't conform to the latest specification of the user ID
     /// grammar but is still accepted because it was previously allowed.
     pub fn is_historical(&self) -> bool {
-        !localpart_is_fully_conforming(self.localpart()).unwrap()
+        !localpart_is_fully_conforming(self.localpart()).unwrap_or(false)
```

Comment:
Please undo this change. When there is a library bug it is better to panic than to return a value.